mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -1,3 +1,4 @@
+from functools import reduce
```

Comment:
where is it called?

Reply:
It used to be called with a previous method, removed now